### PR TITLE
OTValidate

### PR DESCRIPTION
**Pull request by randy-seng**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -101,6 +102,11 @@ extra_css:
   - stylesheets/color.css
   - stylesheets/fonts.css
 
+extra_javascript:
+  - assets/js/mathjax.js
+  - https://polyfill.io/v3/polyfill.min.js?features=es6
```
Comment (frunika) [inline mkdocs.yml:107]: no external js for privacy reasons ;)

Comment (frunika) [reply mkdocs.yml:107]: why do we need polyfill?

Comment (randy-seng) [reply mkdocs.yml:107]: Oh I see! Apparently it is needed to enable [mathjax](https://squidfunk.github.io/mkdocs-material/reference/mathjax/). In other words to enable LaTeX math notation.

Comment (frunika) [reply mkdocs.yml:107]: Okay, so we need to use the external sources. I think the two sources can be considered trustworthy.

**Review by frunika — APPROVED** [review]

**Pull request merged**